src/components/Circular Support Indicator/CircularSupportIndicator.scss

```diff
@@ -0,0 +1,63 @@
+@use "./variables";
+
```

Comment:
@tobiastykvart wanted to ask, so are we using CSS, SCSS or MUI styling? I think we should stick to one and keep consistency

Reply:
The end goal is to use MUI themes to style the MUI components we are using, and [Emotion](https://emotion.sh/docs/introduction) or the `sx` prop from MUI to do any additional styling.
Writing a separate .scss and importing it works as well but I think we want to move away from that.
This was copy/pasted from the SupportIndicator I developed but I'll delete this file since it's not being used anyway.